vendor/init.bat

```diff
@@ -220,70 +292,71 @@ exit /b
     if %verbose-output% gtr 0 echo %*
     exit /b
 
+:show_error
+    echo ERROR: %*
+    echo CMDER Shell Initialization has Failed!
+    exit /b
+
+:run_profile_d
+  if not exist "%~1" (
+    mkdir "%~1"
+  )
+  
+  pushd "%~1"
+  for /f "usebackq" %%x in ( `dir /b *.bat *.cmd 2^>nul` ) do (
+    call :verbose-output Calling "%~1\%%x"...
+    call "%~1\%%x"
+  )
+  popd
+  exit /b
+
 ::
 :: specific to git version comparing
 ::
 :read_version
-
     :: clear the variables
     set GIT_VERSION_%~1=
 
     :: set the executable path
     set "git_executable=%~2\git.exe"
+    call :verbose-output git_executable=%git_executable%
 
     :: check if the executable actually exists
     if not exist "%git_executable%" (
-        :: return a negative error code if the executable doesn't exist
+        call :verbose-output "%git_executable%" does not exist!
         exit /b -255
     )
 
     :: get the git version in the provided directory
-    for /F "delims=" %%F in ('%git_executable% --version 2^>nul') do @(
-        set "GIT_VERSION_%~1=%%F"
+    for /F "tokens=1,2,3 usebackq" %%F in (`"%git_executable%" --version 2^>nul`) do @(
```

Comment:
Yes, my mistake. Should have used `usebackq` with that syntax to avoid paths with space in their name. Damn you, MS Windows!